typing_tester.py

```diff
@@ -0,0 +1,114 @@
+import curses
+from curses import wrapper
+import time
+
+# To calcukate accuracy
+def cal_accuracy(given_word, user_text, backspaces):
```

Comment:
Read pep8 conventions and figure out what are docstrings, how to used them and how they are effective